### PR TITLE
Cleanup plugin.xml and Gradle

### PR DESCRIPTION
**Pull request by rock3r**
A few minor adjustments (see commits list). Requires #64 to be merged first.

Actual diff: https://github.com/rock3r/kotlin-fill-class/compare/trailing-comma...rock3r:cleanup

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlinVersion = '1.6.0'
+    ext.kotlinVersion = '1.6.20'
```
Comment (rock3r) [inline build.gradle:2]: IJ 2022.1 bundles Kotlin 1.6.20

File: build.gradle [hunk 2]
```diff
@@ -54,7 +53,11 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     testImplementation group: 'junit', name: 'junit', version: '4.13.1'
 
-    ktlint "com.github.shyiko:ktlint:0.31.0"
+    ktlint("com.pinterest:ktlint:0.45.2") {
```
Comment (rock3r) [inline build.gradle:56]: Updated ktlint usage according to its readme

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -10,12 +10,12 @@
...
 
     <change-notes><![CDATA[
-    1.0.12 - Add support for 221* / 2022.1 version of IDEA
+    1.0.13- Add trailing comma support
     ]]>
     </change-notes>
 
     <!-- please see https://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
-    <idea-version since-build="202" until-build="213.*"/>
+    <idea-version since-build="191.8026.42" until-build="221.*"/>
```
Comment (rock3r) [inline src/main/resources/META-INF/plugin.xml:18]: Aligned to actual values (may consider removing this from here or from the build.gradle, to make it easier to keep up to date)

Comment (suusan2go) [reply src/main/resources/META-INF/plugin.xml:18]: agree. happy to remove these 👍

**Comment by suusan2go** [comment]
@rock3r Thanks for cleaning up! My apology I couldn't have time to look into this earlier. You might need to change `kotlinVersion` and `version`,  but overall looks good to me 👍 Let me know if you are ready for review.

**Comment by rock3r** [comment]
No problems for the timing, I understand sometimes it's very hard to keep the balance as is :)

**Comment by suusan2go** [comment]
@rock3r before merging this, can you rebase the PR with the latest master?

**Comment by rock3r** [comment]
I can but not before Friday as I am on vacation

**Comment by rock3r** [comment]
Had totally forgotten I can do it from the website 😃

**Comment by suusan2go** [comment]
@rock3r Thank you for your contribution!

**Comment by rock3r** [comment]
Happy to help!

**Pull request merged**